package chapter10.replaceExceptionwithTest.after;

/**
 * Created by simjunbo on 2018-09-17.
 */
public class Resource {
}
